packages/fxa-payments-server/server/lib/amplitude.js

```diff
@@ -38,13 +49,33 @@ const transform = initialize(
   FUZZY_EVENTS
 );
 
-module.exports = (event, request, data) => {
+// TODO: remove eslint ignore in FXA-6950
+// eslint-disable-next-line no-unused-vars
```

Comment:
I added this for each helper, as they aren't used anywhere yet, but they will be in FXA-6950 which will follow this patch.